### PR TITLE
Add sizing helpers for SVGs

### LINKED ISSUE #64: Icons
Icons for general usage. Various sizes will be necessary.

For use in various locations in apps/sites

### PR DESCRIPTION
**Pull request by connor-baer**
**Changes**

- Add sizing helpers for SVGs

Closes #64.

### CODE REVIEW

**Comment by mlent** [comment]
Are these style helpers also exported for users of the component library?

**Comment by connor-baer** [comment]
@mlent They are: https://github.com/sumup/circuit-ui/blob/master/src/styles/index.js#L4

File: src/styles/style-helpers.js [hunk 1]
```diff
@@ -53,6 +53,13 @@ export const textKilo = createTypeHelper('text', 'kilo');
 export const textMega = createTypeHelper('text', 'mega');
 export const textGiga = createTypeHelper('text', 'giga');
 
+/**
+ * SVGs
+ */
+
+export const svgKilo = size('16px');
```
Comment (felixjung) [inline src/styles/style-helpers.js]: Can this maybe use the variable from the theme? Probably call it with the theme as argument. The function then picks the according spacing from the theme.

Comment (mlent) [reply src/styles/style-helpers.js]: I don't think this is actually a spacing though. Technically it's an SVG size, and not tied semantically to spacing IMO.

**Pull request merged**